kumascript/src/api/mdn.ts

```diff
@@ -1,10 +1,13 @@
 import got from "got";
 import { KumaThis } from "../environment";
 import * as util from "./util";
+import { INTERACTIVE_EXAMPLES_BASE_URL } from "../../../libs/env";
 
 // Module level caching for repeat calls to fetchWebExtExamples().
 let webExtExamples: any = null;
 
+let interactiveExampleHeightData: object = null;
```

Comment:
```suggestion
type InteractiveExamplesHeight = 'shorter' | 'taller' | 'tabbed-shorter' | 'tabbed-standard' | 'tabbed-taller';
type InteractiveExamplesHeightData = Record<string, InteractiveExamplesHeight>;
let interactiveExampleHeightData: InteractiveExamplesHeightData = null;
```

Reply:
I added the correct type instead of the object. I have moved it to interactive-example.tsx though, so the whole logic is located in a single place. In the future I will port BOB to TS, so we can remove the types from here.